frontend/wasm/src/api.rs

```diff
@@ -320,8 +320,16 @@ impl JsEditorHandle {
 	}
 
 	/// A font has been downloaded
-	pub fn on_font_load(&self, font_file_url: String, data: Vec<u8>, is_default: bool) -> Result<(), JsValue> {
-		let message = DocumentMessage::FontLoaded { font_file_url, data, is_default };
+	pub fn on_font_load(&self, font_file_url: String, data: Vec<u8>) -> Result<(), JsValue> {
+		let message = DocumentMessage::FontLoaded { font_file_url, data };
+		self.dispatch(message);
+
+		Ok(())
+	}
+
+	/// The default font has been downloaded
+	pub fn on_default_font_load(&self, font_file_url: String, data: Vec<u8>) -> Result<(), JsValue> {
```

Comment:
This basically does the same thing as `on_font_load()`. Is there a reason we need to distinguish between the default and any ordinary font load request? Why can't we just send the editor a font with `DocumentMessage::FontLoaded` and let it cache that?

Reply:
This one sends a portfolio message that populates the state in the portfolio

Reply:
Shouldn't all loaded fonts populate the cache the portfolio? It seems weird to me that we'd only have the default font do that.

Reply:
So we no longer store the font cache in the Document?

Reply:
This would be quite a large refactor.

Reply:
I'm inclined to answer that it doesn't make sense to have a per-document font cache. It seems like an editor-centric concept, since you'll have many documents which would benefit by sharing the font. But perhaps I'm overlooking arguments in favor of it being per-document? Happy to discuss this further and work out what's best.

Reply:
> This would be quite a large refactor.

We can avoid doing that refactor now if you'd prefer (let's just add a TODO comment and probably file an issue for it). Is there an intermediary state where this code can integrate the comments I gave (to generally clean up/generify the concept of the default font) without doing that full refactor?